[2022-06-02 09:12:59,485] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first scheduled__2022-05-31T00:00:00+00:00 [queued]>
[2022-06-02 09:12:59,505] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first scheduled__2022-05-31T00:00:00+00:00 [queued]>
[2022-06-02 09:12:59,506] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 09:12:59,507] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-06-02 09:12:59,508] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-06-02 09:12:59,536] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): run_this_first> on 2022-05-31 00:00:00+00:00
[2022-06-02 09:12:59,540] {standard_task_runner.py:52} INFO - Started process 72 to run task
[2022-06-02 09:12:59,546] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'todoist_load_to_pg', 'run_this_first', 'scheduled__2022-05-31T00:00:00+00:00', '--job-id', '398', '--raw', '--subdir', 'DAGS_FOLDER/todoist_load_to_pg.py', '--cfg-path', '/tmp/tmp2aeykuow', '--error-file', '/tmp/tmpwun4mvln']
[2022-06-02 09:12:59,552] {standard_task_runner.py:77} INFO - Job 398: Subtask run_this_first
[2022-06-02 09:12:59,663] {logging_mixin.py:109} INFO - Running <TaskInstance: todoist_load_to_pg.run_this_first scheduled__2022-05-31T00:00:00+00:00 [running]> on host bd658c040114
[2022-06-02 09:12:59,797] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=emil
AIRFLOW_CTX_DAG_ID=todoist_load_to_pg
AIRFLOW_CTX_TASK_ID=run_this_first
AIRFLOW_CTX_EXECUTION_DATE=2022-05-31T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-31T00:00:00+00:00
[2022-06-02 09:13:00,801] {python.py:175} INFO - Done. Returned value was: None
[2022-06-02 09:13:00,821] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=todoist_load_to_pg, task_id=run_this_first, execution_date=20220531T000000, start_date=20220602T091259, end_date=20220602T091300
[2022-06-02 09:13:00,863] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-02 09:13:00,912] {dagrun.py:627} WARNING - Failed to get task '<TaskInstance: todoist_load_to_pg.task_api_active scheduled__2022-05-31T00:00:00+00:00 [None]>' for dag 'todoist_load_to_pg'. Marking it as removed.
[2022-06-02 09:13:00,920] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
